### DISCUSSION TITLE
3D printed dust covers for cart slots?

### DISCUSSION THREAD

**Discussion by Veridor** [sanni/cartreader#641] [Ideas & Suggestions]
Are there files for 3D printed dust covers for the v5 cart slots? It seemed like it in this picture, or am I mistaken? 

https://camo.githubusercontent.com/a177d1f08dd67a6da35b067d632b50affe0a2f1509007ead82b0239b14c80de8/68747470733a2f2f646c2e64726f70626f7875736572636f6e74656e742e636f6d2f732f6439333739767636797a31746e35702f687735776672616d6569736f2e6a70673f646c3d31

**Comment by sanni** [comment]
The boxes in the picture are meant to represent the cart slots. I was just to lazy to model the pins inside. Labeled dust covers would be a nice idea though.

**Reply by Cephiros** [reply]
indeed. there are these until then: https://www.thingiverse.com/thing:4529471
